api/src/main/java/com/alibaba/nacos/api/naming/pojo/ServiceInfo.java

```diff
@@ -68,18 +69,22 @@ public void setAllIPs(boolean allIPs) {
     }
     
     public ServiceInfo(String key) {
```

Comment:
I think it's better to comment on each case here

Reply:
> I think it's better to comment on each case here

OK, I wll add it